lib/ansible/config/base.yml

```diff
@@ -318,6 +318,19 @@ COLOR_WARN:
   env: [{name: ANSIBLE_COLOR_WARN}]
   ini:
   - {key: warn, section: colors}
+CONDITINAL_BARE_VARS:
```

Comment:
Nobody noticed the typo here?

It lacks an O in CONDITIONAL. Tests should fail, or I missed something.

Reply:
its a typo, but it is consistently typoed on consumption also

Reply:
In `conditional.py`, yes, but the test runs the playbook with `ANSIBLE_CONDITIONAL_BARE_VARS`. Is it ok?

Reply:
why wouldn't it be?  That is the 'env variable' , which is not the thing that was misspelled

Reply:
fixed in #53809